client/src/components/Ratings/sub-components/ReviewModal.jsx

```diff
@@ -29,26 +29,39 @@ export default function ReviewModal({
     }
   };
 
+  const closeModal = () => {
+    document.querySelector('.write-review-modal').classList.remove('denial');
+    document.querySelector('.write-review-modal').classList.add('exiting');
+    setTimeout(() => {
+      setReviewModal(false);
+    }, 200);
+  };
+  // Handler Series sets newReview with value of input fields //
   const starRatingTextHandler = (value) => {
     if (value === 1) {
       setStarRatingText('Poor');
       setNewReview({ ...newReview, rating: value });
+      document.querySelector('.review-asterisk-stars').classList.remove('red');
     }
     if (value === 2) {
       setStarRatingText('Fair');
       setNewReview({ ...newReview, rating: value });
+      document.querySelector('.review-asterisk-stars').classList.remove('red');
     }
     if (value === 3) {
       setStarRatingText('Average');
       setNewReview({ ...newReview, rating: value });
+      document.querySelector('.review-asterisk-stars').classList.remove('red');
     }
     if (value === 4) {
       setStarRatingText('Good');
       setNewReview({ ...newReview, rating: value });
+      document.querySelector('.review-asterisk-stars').classList.remove('red');
     }
     if (value === 5) {
       setStarRatingText('Great');
       setNewReview({ ...newReview, rating: value });
+      document.querySelector('.review-asterisk-stars').classList.remove('red');
     }
```

Comment:
For these,
       setNewReview({ ...newReview, rating: value });
       document.querySelector('.review-asterisk-stars').classList.remove('red');

I think you can refactor the repeated code outside the if blocks.

Reply:
I agree this can be re factored. Built it crudely in interest of time.